js/levels.js

```diff
@@ -52,7 +52,7 @@ function createForestLevel() {
 function createDesertLevel() {
   return new Level("Desert", [
     [desert.none, desert.right, desert.right, desert.right, desert.right, desert.none],
-    [desert.none, desert.top, desert.both, desert.both, desert.both, desert.both],
+    [desert.drawnbyme, desert.top, desert.both, desert.both, desert.both, desert.both],
```

Comment:
this will break I guess, since you added it to door.drawnbyme, it is not available for desert.drawnbyme.